integrations/influxdb_connector/influxdb-timestream-connector/README.md

```diff
@@ -55,10 +55,18 @@ The following parameters are available when deploying the connector as part of a
 
 | Parameter     | Description | Default Value |
 |---------------|-------------|---------------|
+| `CustomPartitionKeyDimension` |  The dimension to use as the partition key. This parameter is required if the CustomPartitionKeyType parameter is set to `dimension`. | |
+| `CustomPartitionKeyType` | The type of custom partition key to use. Valid options are `dimension` or `measure`. The `dimension` option requires the CustomPartitionKeyDimension parameter to also be set. If this parameter is not provided, newly-created tables will use default partitioning and none of the parameters relating to custom partition keys will be used. | |
```

Comment:
> If this parameter is not provided, ...
Is this behavior useful and expected? The previous sentence implies that missing `dimension` should result in a validation error.

Reply:
Whether `CustomPartitionKeyType` has a value determines whether the other partition key environment variables are used or ignored.

`dimension` is an "option" in the sense that it is one possible value for `CustomPartitionKeyType`. Depending on whether `CustomPartitionKeyType` is `dimension` or `measure`, other environment variables need to be set.

Reply:
What happens if the other variables are not set?

Reply:
If `CustomPartitionKeyType` is `measure`, no other environment variables need to be set. If `CustomPartitionKeyType` is `dimension` then we do a check of these other environment variables and return an `Err` if any are missing.

If we didn't do a check, we would eventually get an error when attempting to create a table.

I noticed that the error messages in one of our checks is inaccurate, I have fixed this.